test/db/tools/rz

```diff
@@ -98,6 +98,7 @@ EOF
 RUN
 
 NAME=no-nl-at-eof script
+BROKEN=1
```

Comment:
Why?

Reply:
because it does work on my linux but not on the linux machine via the CI and i have no clue why